libs/blocks/iframe/iframe.js

```diff
@@ -48,6 +50,15 @@ export default function init(el) {
   const iframe = createTag('iframe', { src: linkHref, allowfullscreen: true });
   const embed = createTag('div', { class: `milo-iframe ${classes}` }, iframe);
 
+  iframe.onload = () => {
+    if ((new URL(iframe.src).origin !== window.location.origin) && ariaLabel) {
```

Comment:
Just wanted to double check: if the first condition - (new URL(iframe.src).origin !== window.location.origin) - is true, but there is no ariaLabel on the anchor, on the line 59 we will get an error due to CORS. Is it guaranteed that an anchor will always have some ariaLabel?